README.adoc

```diff
@@ -249,6 +395,15 @@ go get -u github.com/kisielk/errcheck
 | `make docker_push_master`     | Pushes master image to Docker image hosting repository
 |===
 
+
+=== Using Postman to test the REST API
+
+You can install the https://www.getpostman.com/[Postman] tool which will be useful to call and test the REST API endpoints of this server.
+Following an image to show how it works.
+
+image::https://user-images.githubusercontent.com/7708031/53174794-8878a600-35e2-11e9-889d-79973d9e16db.png[Postman]
+
+
 == Built With
```

Comment:
@camilamacedo86  would this make more sense as a badge at the top of the page?

Reply:
@finp @camilamacedo86 I'm not sure if Postman should be in the README. It's an external tool for making REST calls but it is nothing to do with this project. Generally, if somebody wants to work on a project like this they would know what Postman is and they may prefer to use something else (their IDE, CURL etc). I don't think a screenshot of Postman should be in here showing them how to use it - it is pretty straightforward anyway and it only serves to make our documentation more busy than it should be. What do you think?

Reply:
@craicoverflow the idea here is to have in the documented how a dev can contribute with and some tips.